deno_webgpu/webidl.rs

```diff
@@ -680,3 +680,138 @@ pub fn features_to_feature_names(
 
   return_features
 }
+
+#[derive(Clone, Copy)]
+pub(crate) struct GPUTextureUsageFlags(pub(crate) wgpu_types::TextureUsages);
+
+impl<'a> WebIdlConverter<'a> for GPUTextureUsageFlags {
```

Comment:
Certainly doing it this way is better than performing ad-hoc fallible conversions elsewhere. But these three converters are pretty repetitive. If there are other flag-like types that would benefit from this treatment, it might be worth figuring out some way to do this with generic code, or at least a macro.